plugins/basic/menu.py

```diff
@@ -0,0 +1,39 @@
+import inspect
+import os
+import sys
+
+
+def plugin_startup():
+    """
+    Initializes the Toolkit plugin for Nuke.
+    """
+
+    current_file_path = os.path.abspath(
+        inspect.getsourcefile(lambda: 0)
+    )
+
+    plugin_root_path = os.path.dirname(current_file_path)
+
+    # the plugin python path will be just below the root level. add it to
+    # sys.path
+    plugin_python_path = os.path.join(plugin_root_path, "python")
+    sys.path.insert(0, plugin_python_path)
+
+    # now that the path is there, we can import the plugin bootstrap logic
+    try:
+        from tk_nuke_basic import plugin_bootstrap
```

Comment:
i think this will prevent certain tk reloads to work. might be worth poining out in comments